styles/adoptadog-stylesheet.css

```diff
@@ -0,0 +1,305 @@
+/* This trick removes any default browser styles and only applies margins explicitly
+set by your css. For me, Chrome always adds a margin to the body by default,
+ruining some of the header and footer styles.*/
+*{margin:0;}
+
+header {
+  color: white;
+  background: linear-gradient(to right, white, purple, gold, white);
+  background-image: url('../images/header-image.jpg');
+  padding: 3rem;
```

Comment:
The padding is affecting the size of body and therefore other elements. Especially noticeable at smaller resolutions. I would remove it.